### PR TITLE
Limit errors sent per page load

### PR DESCRIPTION
**Pull request by monde**
@josh here's my implementation for issue `Add option to limit errors sent per page load` #79. Let me know if the naming of functions and variables are acceptable, the commits in the PR are clean and I can easily rebase in name changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ All notable changes to this project will be documented in this file. See [Keep a
 CHANGELOG](http://keepachangelog.com/) for how to update this file. This project
 adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [0.5.2] - 2017-10-18
```
Comment (joshuap) [inline CHANGELOG.md]: Let's keep the unreleased tag for now--we'll do a separate commit on master when we release (which will then be tagged with the version number).

File: README.md [hunk 2]
```diff
@@ -157,12 +157,20 @@ Honeybadger.configure({
   disabled: false,
 
   // Send notifications asynchronously
-  async: true
+  async: true,
+
+  // Limit the number of errors the client will send to Honeybadger after
+  // page load. Default is unlimited (undifined)
```
Comment (joshuap) [inline README.md]: Typo: undefined

**Comment by monde** [comment]
I'll make the changes and rebase a cleaned up PR

**Comment by monde** [comment]
@joshuap the PR is cleaned up and builds are 💚💚💚💚💚💚💚

**Comment by joshuap** [comment]
🎉 thanks! 👍 👍
